code/modules/mob/living/carbon/human/life.dm

```diff
@@ -81,6 +81,19 @@
 		to_chat(src, "<span class='notice'>You don't feel like harming anybody.</span>")
 		a_intent_change(INTENT_HELP)
 
+	GET_COMPONENT_FROM(mood, /datum/component/mood, src)
+	if (getBrainLoss() >= 60 && stat == CONSCIOUS)
```

Comment:
isn't there a trait for brain damage you can piggyback off?

Reply:
I will go check.

Reply:
apparently checking brain damage directly is better because traumas aren't guaranteed by brain damage.